CONTRIBUTING.md

```diff
@@ -651,3 +651,79 @@ Names of files and directories should be in lowercase, with dashes between words
+
+The code artefacts are the hardest for committers to assess because PRs touch all sorts of components: applications, libraries, NixOS modules, editor plugins and many many other things. Any individual committer can only really assess components that they themselves know how to use however and yet they must still be convinced somehow. There isn't a good generic solution to this but there are some ways easing the committer's job here:
+
+- Provide smoke tests that the committer can run without much research or setup.
+  Committers usually don't have the time or interest to learn how your component works and how they could test its functionality. If you can provide a quick guide on how to use the component in a meaningful way or a ready-made command that demonstrates that the package works as expected, the committer can easily convince themselves that your change is good.
+  If it's automate-able, you could even turn this smoke test into an automated NixOS test which reviewers could simply run via Nix.
+- Invite other users of the component to try it out and report their findings.
+  If a committer sees the testimonials of other users trying your change and it works as expected for them, that too can convince the committer of your PR's quality.
+- Describe what you have done to test your PR.
+  If you can convince the committer that you have done sufficient quality assurance on your changes and they trust your report, this too can convince them of your PR's quality, albeit not as strongly as the methods above.
+- Be a maintainer of the component.
+  This isn't something you can do on your first few PRs touching a component but listed maintainers generally receive more trust when it comes to changes to their maintained components and committers may opt to merge changes without deeper review when they see they're done by their respective maintainer.
+
+Even if you adhere to all of these recommendations, it is still quite possible for your PR to be forgotten or abandoned by any given committer. Please remain mindful of the fact that they are doing this on their own volition and unpaid in their free time and therefore [owe you nothing](https://mikemcquaid.com/open-source-maintainers-owe-you-nothing/). Causing a stink in such a situation is a surefire way to get any other potential committer to not want to look at your PR either.
+Ask them nicely whether they still intend to review your PR and find yourself another committer to look at your PR if not.
+
+### How can I get a committer to look at my PR?
+
+- Simply waiting. Reviewers frequently browse open PRs and may happen to run across yours and take a look.
```

Comment:
```suggestion
- Simply wait.

  Reviewers frequently browse open PRs and may happen to run across yours and take a look.
```

Use imperative in instructions. Gerund is just more letters, and thus, more to parse.